EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001258	If we destroy that sinister idol...{nl}There might be a good chance that the whole wall may disappear.
+QUEST_LV_0300_20160310_001259	I just don't know how I can destroy it...{nl}There is nothing I can think of at the moment.{nl}
+QUEST_LV_0300_20160310_001260	No, wait...{nl}I can sense the demons' souls from you.{nl}
+QUEST_LV_0300_20160310_001261	That sphere you have...{nl}Is a demon's soul locked inside it?{nl}
+QUEST_LV_0300_20160310_001262	That's it.{nl}Maybe we can use this to destroy the sinister idol.{nl}
+QUEST_LV_0300_20160310_001263	The idea is to purify the demons' souls and use their power.{nl}Unfortunately, we lack the right amount of souls to destroy the idol.{nl}
+QUEST_LV_0300_20160310_001264	Can you fill it with a little bit more of the demons' souls?{nl}As you already may know, defeating demons will automatically absorb their souls.
+QUEST_LV_0300_20160310_001265	The demons are strong so... I guess that their souls are strong as well.{nl}My sacred energy is not strong enough to purify this much power.
+QUEST_LV_0300_20160310_001266	We should somehow borrow the power of holy water...{nl}Like I told you before though, I have nothing at the moment.
+QUEST_LV_0300_20160310_001267	I am so sorry, but can you get me Holy Water from the Priest Master in Klaipeda?{nl}If we have enough holy water, we'll be able to purify the demons' fallen souls.
+QUEST_LV_0300_20160310_001268	Please use the statue of Goddess Vakarine near Viyme Fountain.{nl}You can quickly come back from Klaipeda.
+QUEST_LV_0300_20160310_001269	With this, you'll be able to purify the souls contained within the Soul Sphere.{nl}Now, give me a moment.
+QUEST_LV_0300_20160310_001270	Since the power of a demon's soul is strong, the residual power after purification possesses strength as well.{nl}This power has the capability to destroy the sinister idol.{nl}
+QUEST_LV_0300_20160310_001271	Put the purified Soul Sphere's power into the sinister idol.{nl}I hope the idol can't endure this...
+QUEST_LV_0300_20160310_001272	Finally!{nl}Now, we can be freed from the black wall.{nl}
+QUEST_LV_0300_20160310_001273	It's been such a long time...{nl}I think my prayer to the goddess was not in vain...
+QUEST_LV_0300_20160310_001274	Now, we'll need to focus on the main culprit of the Tree of Truth sect, Dominikas...{nl}Everything will be over once we drive him out.{nl}
+QUEST_LV_0300_20160310_001275	Please lend me your power a little longer.{nl}Even if we have the methods to catch him, we can't use our powers {nl}since the villagers have been possessed from being tortured so long...{nl}
+QUEST_LV_0300_20160310_001276	He seems to have some power to control the plants freely.{nl}I have to warn you because he is not to be underestimated.
+QUEST_LV_0300_20160310_001277	Dominikas... He ran away...{nl}
+QUEST_LV_0300_20160310_001278	We shouldn't worry too much.{nl}It's a big harvest that we drove out the Tree of Truth sect from Elgos Monastery.{nl}
```

Comment:
"It's a big harvest that we drove out" is weird. Convo in discord - will add solution soon.